diff --color -Nur calibre-6.17.0.orig/src/calibre/linux.py calibre-6.17.0/src/calibre/linux.py
--- calibre-6.17.0.orig/src/calibre/linux.py	2023-05-06 11:36:35.703460982 -0700
+++ calibre-6.17.0/src/calibre/linux.py	2023-05-06 11:41:24.059853007 -0700
@@ -1094,6 +1094,7 @@
 TryExec=lrfviewer
 Exec=lrfviewer %f
 Icon=calibre-viewer
+NoDisplay=true
 MimeType=application/x-sony-bbeb;
 Categories=Office;Viewer;
 '''
@@ -1108,6 +1109,7 @@
 TryExec=ebook-viewer
 Exec=ebook-viewer --detach %f
 Icon=calibre-viewer
+NoDisplay=true
 Categories=Office;Viewer;
 '''
 
@@ -1121,6 +1123,7 @@
 TryExec=ebook-edit
 Exec=ebook-edit --detach %f
 Icon=calibre-ebook-edit
+NoDisplay=true
 Categories=Office;WordProcessor
 '''
 
